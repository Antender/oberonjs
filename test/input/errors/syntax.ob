MODULE m1;
VAR i: INTEGER;
BEGIN
	i := 1; (* extra semicolon *)
END m1.