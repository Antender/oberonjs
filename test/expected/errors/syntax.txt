line 4: END expected (MODULE)
